packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -241,6 +246,15 @@ class SelectableRegion extends StatefulWidget {
   /// Called when the selected content changes.
   final ValueChanged<SelectedContent?>? onSelectionChanged;
 
+  /// {@template flutter.widgets.SelectableRegion.separator}
+  /// The separator to use when concatenating the selected text. Defaults to an
```

Comment:
The first paragraph should be a single sentence.  So add a line break after the first sentence and an empty line of `///` before "Defaults"